node/flags/flags.go

```diff
@@ -231,6 +231,13 @@ var (
 		EnvVar:   common.PrefixEnvVar(EnvVarPrefix, "ONCHAIN_STATE_REFRESH_INTERVAL"),
 		Value:    1 * time.Hour,
 	}
+	ChunkDownloadTimeoutFlag = cli.DurationFlag{
+		Name:     common.PrefixFlag(FlagPrefix, "chunk-download-timeout"),
+		Usage:    "The timeout for downloading chunks from the relay (default: 30s)",
+		Required: false,
+		EnvVar:   common.PrefixEnvVar(EnvVarPrefix, "CHUNK_DOWNLOAD_TIMEOUT"),
+		Value:    30 * time.Second,
+	}
```

Comment:
The default timeout on the relay side of the fence is 20 seconds (`relay/cmd/flags/flags.go: GetChunksTimeoutFlag`). I don't really care what value we use, but we should probably assign the same value for both settings.